### PR TITLE
Updates on MUSIC:

### PR DESCRIPTION
**Pull request by chunshen1987**
(1) Use the specialized MUSIC version for JETSCAPE (more than 2x boost in speed)
     This specialized version is developed for the XSCAPE project and has backward compatibility.
(2) Add a user xml to run Trento + MUSIC + iSS
(3) Update parameter names in the MUSIC wrapper

### CODE REVIEW

**Comment by jhputschke** [comment]
Hi Chun, just to clarify, you want this to get merged in the Jetscape and also Xscape repo? If this is for jetscape mainly a speed improvement can we try to do this is in the 3.6 RC?

**Comment by chunshen1987** [comment]
This pull request is just the JETSCAPE repo. The XSCAPE framework is already using this version of MUSIC.

Yes, feel free to merge to the 3.6 RC if it is in time. 

Chun

File: config/jetscape_user_MUSIC.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        </LongiInputs>
+        </Trento>
+    </IS>
+
+  <!-- Hard Process -->
+  <Hard>
+    <PythiaGun>
+      <pTHatMin>50</pTHatMin>
+      <pTHatMax>70</pTHatMax>
+      <eCM>5020</eCM>
```
Comment (jhputschke) [inline config/jetscape_user_MUSIC.xml:46]: Hi Chun in any case if this is the example xml for RHIC we should change pThat bin and sqrts in. Pythia too, maybe 20-30 and 200 ...

**Comment by jhputschke** [comment]
Also I don't see how I could change the PR to the 3.6 RC branch if we want to merge this PR with the upcoming one ...

**Comment by chunshen1987** [comment]
I generated a separate pull request for 3.6 RC.

File: config/jetscape_user_MUSIC.xml (hunk 1 above)
Comment (chunshen1987) [reply config/jetscape_user_MUSIC.xml]: Updated.

**Comment by latessa** [comment]
Since this content was merged into the 3.6 RC in a separate pull request (#164), can this pull request (#163) be closed without merging?

**Pull request closed without merging**